### ISSUE TITLE
Bug when doing env.copy()

### ISSUE THREAD

**Issue by youkaichao** [BDonnot/lightsim2grid#10]
## Environment
 - Grid2op version: `1.1.1`
 - System: `ubuntu16.04, ...`
 - Additional system information

## Bug description

When using lightsim2grid as the backend, environments cannot be copied.

### Code snippet

```python
from grid2op import make
from lightsim2grid import LightSimBackend as backend
env = make('l2rpn_case14_sandbox', backend=backend())
env.copy()
```

## Current output
```
TypeError: cannot pickle 'lightsim2grid_cpp.GridModel' object
```

**Comment by BDonnot** [comment]
This has been fixed in the latest release.

Make sure to update grid2op to version 1.2.1 and lightsim to version 0.2.4

**Comment by youkaichao** [comment]
I have updated to the latest version but it still does not work :( 

Seems like some attributes are not correctly copied.

grid2op version 1.2.1 and lightsim version 0.2.4

```python
from grid2op import make
from lightsim2grid import LightSimBackend as backend
env = make('l2rpn_case14_sandbox', backend=backend())

for i in range(10):
    env.step(env.action_space({}))

env1 = env.copy()

done = False
cnt = 0
while not done:
    o, r, done, info = env.step(env.action_space({}))
    cnt += 1
print(cnt)

done = False
cnt = 0
while not done:
    o, r, done, info = env1.step(env1.action_space({}))
    cnt += 1
print(cnt)


.........
    360         """
--> 361         volt_control_act = self.voltage_controler.fix_voltage(self.current_obs,
    362                                                               agent_action,
    363                                                               self.env_modification,

AttributeError: 'NoneType' object has no attribute 'fix_voltage'
```

**Comment by BDonnot** [comment]
Hello,

I reopened the issue.
By the way, this is why we ask for reproducible code. You put the code of the issue in a test, and fix grid2op / lightsim until it's working. 
The code you put in the first step was working, so the issue you posted was fixed.
But you didn't posted the piece of code you wanted to run :-/

I hope this time this will be ok. I am including it as a new grid2op version (were this issue should have been posted) that will be released ASAP.
